aws/terminator/__init__.py

```diff
@@ -270,7 +270,7 @@ def is_vpc_default(self, vpc_id: str) -> bool:
 class DbTerminator(Terminator):
     """Base class for classes which find and terminate AWS resources with age tracked via DynamoDB."""
     def __init__(self, client: botocore.client.BaseClient, instance: typing.Dict[str, typing.Any]):
-        super(DbTerminator, self).__init__(client, instance)
+        super().__init__(client, instance)
```

Comment:
It would be nice to review and discuss the change to `tox` independently of the code fixes in `aws/terminator/`, which shouldn't need much discussion. Could you put this, and the other fixes, into a separate PR?

Reply:
I've just opened this PR with the Python changes. It has to be merged before this PR or the tests will fail: https://github.com/mattclay/aws-terminator/pull/246